pyblish_lite/i18n/zh_CN.ts

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS><TS version="1.1" language="zh_CN">
+<context>
+    <name>Window</name>
+    <message>
+        <location filename="../window.py" line="763"/>
+        <source>Finishing up reset..</source>
+        <translation>完成重置..</translation>
+    </message>
+    <message>
+        <location filename="../window.py" line="228"/>
+        <source>Comment..</source>
+        <translation>备注..</translation>
+    </message>
+    <message>
+        <location filename="../window.py" line="722"/>
+        <source>Processing</source>
+        <translation>处理</translation>
+    </message>
+    <message>
+        <location filename="../window.py" line="877"/>
```

Comment:
As someone who hasn't written these before, how does one make these? How do we update them? It looks like a line number in there, does that mean that if we refactor the code, we should update this file? Or is this file generated from something else? How does one provide translations for yet another language?

What is the `pyblish_lite/i18n/zh_CN.qm` file? Is there any documentation you could recommend to better understand how this works? Thanks again, so exciting!

Reply:
The `.ts` file is generated by  `pyside-lupdate pysite_lite.pro`.
The line numbers are just reference for translation, will be removed in .qm file.
Qt matches translation with context(class name), not line number.

The `.qm` file is generated by `lrealese <.ts file>` .

Add another language by doing follow, tools can be found in PySide pacakage folder.:
1. Add `<Locale name>.ts` to 'TRANSLATIONS' of `pyblish_lite.pro`, seperate by space.
  Locale name is return value of `PySide.QtCore.QLocale.system().name()` on you target machine.
2. Run `pyside-lupdate pyblish_lite.pro`
3. Open `.ts` file with `linguist`
4. Translate all items in `linguist`.
5. Release by `lingust` -> `File` menu -> `release`, or use `lrealese <.ts file>`



see https://wiki.qt.io/PySide_Internationalization
http://doc.qt.io/qt-5/i18n-source-translation.html
http://doc.qt.io/qt-5/linguist-translators.html